### DISCUSSION TITLE
Map card: markers with same geo coords - how to show overlapping markers

### DISCUSSION THREAD

**Discussion by ildar170975** [home-assistant/frontend#20724] [Dashboard]
Example:
Currently it is possible to show states on markers on Map card:
![image](https://github.com/home-assistant/frontend/assets/71872483/c35cd196-7b50-4d3b-b805-41f3b35beeeb)

Some weather stations provide several parameters - like pressure, humidity, temperature.
Assume wee need to show all of the on a map.
Since all these sensors have same geo coordinates - they are overlapping each other, and only the topmost can be seen.

Would be great to do this:
If two or more entities have same coordinates - show one marker on its coordinates, show other markers nearby connected by lines ("links") to the 1st marker:

![image](https://github.com/home-assistant/frontend/assets/71872483/d98c4e60-59ef-47d9-bb07-b1bd83071b76)

May be someone has another (better) ideas.

And probably this approach may be used for any markers, not only with states...

**Comment by karwosts** [comment]
maybe duplicate w/ #17985 if you want to combine.

**Reply by ildar170975** [reply]
Right, will close this

**Comment by ildar170975** [comment]
Closed in favour of https://github.com/home-assistant/frontend/discussions/17985
